### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MR-Os1**
prisma not saving user if metadata is null

### CODE REVIEW

**Comment by krumware** [comment]
@MR-Os1 is there a specific metadata field being null that caused the issue? I retested with multiple fields being null and did not see the issue, just hoping to make sure I understand the issue cause clearly.

**Comment by MR-Os1** [comment]
in the issue [here](https://github.com/calendso/docker/issues/68) the client tries to add new user and the metadata field in the user database is by default null.
and Prisma shows the error in the link because apparently the metadata field in the database can't be null but it can be an empty json.
it's a temporary fix but if we can find a way to set the default value of Prisma not  be null in the new user creation this could be a really better solution.

**Comment by krumware** [comment]
Can you by chance put something in the troubleshooting section instead? This may be something that needs to be addressed in upstream Calendso, so I don't want it to seem like a setup requirement at this point

**Comment by MR-Os1** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -85,3 +85,4 @@ For more advanced usage, please refer to the git documentation: [https://git-scm
 ## Troubleshooting
 
 * SSL edge termination: If running behind a load balancer which handles SSL certificates, you will need to add the environmental variable `NODE_TLS_REJECT_UNAUTHORIZED=0` to prevent requests from being rejected. Only do this if you know what you are doing and trust the services/load-balancers directing traffic to your service.
+* If you had and issue creating user please check if the metadata field is null change it to empty json ({}) and you will be fine.
```
Comment (krumware) [inline README.md:88]:
```suggestion
* Failed to commit changes: Invalid 'prisma.user.create()': Certain versions may have trouble creating a user if the field `metadata` is empty. Using an empty json object `{}` as the field value should resolve this issue. Also, the `id` field will autoincrement, so you may also try leaving the value of `id` as empty.
```

**Pull request merged**